src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -345,6 +345,49 @@ class HttpGoogleServicesDAO(
     }
   }
 
+  override def createGoogleGroup(groupRef: RawlsGroupRef): Future[Unit] = {
+    val directory = getGroupDirectory
+    val groups = directory.groups
+    retry(when500) {
+      () => Future {
+        val inserter = groups.insert(new Group().setEmail(toGoogleGroupName(groupRef.groupName)).setName(groupRef.groupName.value))
+        blocking {
+          inserter.execute
+        }
+      }
+    }
+  }
+
+  override def addMemberToGoogleGroup(groupRef: RawlsGroupRef, memberToAdd: Either[RawlsUser, RawlsGroup]): Future[Unit] = {
+    val member = memberToAdd match {
+      case Left(member) => new Member().setEmail(toProxyFromUser(memberToAdd.left.get.userSubjectId)).setRole(groupMemberRole)
```

Comment:
Use a different variable name besides `member` here in case it is confused with the previous line.

Also, you can use the pattern-matched variable directly instead of `memberToAdd.left.get`